### DISCUSSION TITLE
Eye Gaze Logic

### DISCUSSION THREAD

**Discussion by YaraAmin** [vladmandic/human#180] [Q&A] [answered]
Most of the time the output of Eye Gaze stuff is not accurate for me. I'm planing on implementing new logic for looking at the lens and looking at the screen, if you have any recommendations for that please let me know. Thanks

**Comment by vladmandic** [comment, marked as answer]
eye gaze depends on relative position of iris within the eye - and that is a really small target to analyze  
plus quite often eyelids obstruct view (either due to blinking or due to normal eye positions such as when gazing downwards, iris is no longer clearly visible from the camera)  

you can train your own high quality model that will zoom and focus on eyes thus creating larger target - but other than that, i don't see what can be done

**Comment by YaraAmin** [comment]
Thank you
